### PR TITLE
PP9-13774 share page download output selection

### PR DESCRIPTION
**Pull request by vsta**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "nswag": "npm run nswag-prepare && nswag run \"src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/nswag.json\" && nswag run \"src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/nswag-frontend.json\" && nswag run \"src/picturepark-sdk-v1-fetch/nswag.json\"",
```
Comment (vsta) [inline package.json:3]: support v1 and v2

File: src/picturepark-sdk-v2-angular/projects/picturepark-sdk-v2-angular-ui/src/lib/shared-module/components/confirm-dialog/confirm-dialog.interface.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ export interface ConfirmOptions {
   title: string;
   message: string;
   options?: {
-    okText: string;
+    okText?: string;
```
Comment (vsta) [inline src/picturepark-sdk-v2-angular/projects/picturepark-sdk-v2-angular-ui/src/lib/shared-module/components/confirm-dialog/confirm-dialog.interface.ts:5]: Remove changes if not needed.

File: src/picturepark-sdk-v2-angular/projects/picturepark-sdk-v2-angular-ui/src/lib/utilities/translations.ts [hunk 3]
```diff
@@ -848,6 +871,30 @@ export let TRANSLATIONS = {
       pt: 'A descarga não pode conter mais de 1000 itens de conteúdo!',
     },
   },
+  LandingDialog: {
+    title: {
+      en: 'Terms of service',
+      de: 'Nutzungsbedingungen',
+      fr: 'Terms of service',
```
Comment (vsta) [inline src/picturepark-sdk-v2-angular/projects/picturepark-sdk-v2-angular-ui/src/lib/utilities/translations.ts:878]: Add missing translations.

File: src/picturepark-sdk-v2-angular/projects/picturepark-sdk-v2-angular/src/lib/utilities/storage-key.enum.ts [hunk 4]
```diff
@@ -4,4 +4,6 @@ export enum StorageKey {
   BasketItems = 'basket-items',
   ShareCookieConsent = 'share-cookie-consent',
   IsLightTheme = 'is-light-theme',
+  ViewMode = 'view-mode',
```
Comment (vsta) [inline src/picturepark-sdk-v2-angular/projects/picturepark-sdk-v2-angular/src/lib/utilities/storage-key.enum.ts:7]: Remove ViewMode from storage keys

**Review by neotek — DISMISSED** [review]

**Pull request merged**